lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -77,3 +77,32 @@ def test_sub_structure(self):
                 self.assertIn(unicode(block.location), blocks['blocks'])
             else:
                 self.assertNotIn(unicode(block.location), blocks['blocks'])
+
+    def test_visible_to_staff_only(self):
```

Comment:
To test `visible_to_staff_only` when it is true, I would alter ToyCourseFactory to have one of the items marked as hidden. Does that sound correct or is there somewhere else I should look?